src/components/profile/profile.js

```diff
@@ -3,16 +3,16 @@ import React, { useEffect, useState } from 'react';
 function Profile() {
     const [profileData, setProfileData] = useState(null);
 
-    useEffect(() => {
-        fetch('api/v1/profile?username=nayem')
-            .then(response => response.json())
-            .then(data => setProfileData(data))
-            .catch(error => console.error('Error fetching profile data:', error));
-    }, []);
+    // useEffect(() => {
+    //     fetch('#')
```

Comment:
Hmm not sure if we don't use the endpoint how it will work 🤔

Reply:
so this logic is the next branch I will be working on, signup and login. auth and to retrieve the profile pic and the login info and so on